lib/src/base-styles/_nx-card.scss

```diff
@@ -0,0 +1,173 @@
+}
+
+.nx-card__call-out {
+  .nx-icon {
+    font-size: 48px;
+  }
+
+  .nx-icon--large {
+    font-size: 64px;
+  }
+}
+
+.nx-call__call-out-text {
+  @include bold();
+
+  font-size: 32px;
+}
+
+// Vertically oriented cards appear in a row container
+.nx-card-container--row {
+  @include container-horizontal;
+
+  .nx-card {
+    display: flex;
+    flex-direction: column;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    margin: 0 $nx-spacing-md 0 0;
+    text-align: center;
+    min-width: 230px;
+    max-width: 330px;
+  }
+
+  .nx-card-header {
+    @include container-vertical;
+    margin-bottom: $nx-spacing-md;
+  }
+
+  .nx-card__call-out {
+    margin-bottom: $nx-spacing-xs;
+  }
+
+  .nx-card__text {
+    margin-bottom: $nx-spacing-l;
+  }
+
+  .nx-card-footer {
+    border-top: 1px solid #d1d6f0;
+    padding-top: $nx-spacing-l;
+    width: 100%;
+  }
+}
+
+// Horizontally orientated cards appear in a column
+.nx-card-container--column {
+  @include container-vertical;
+
+  flex-direction: column;
+
+  .nx-card {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: auto 1fr auto;
+    grid-template-columns: auto 1fr auto;
+    -ms-grid-rows: auto auto;
+    grid-template-areas:
+      "card-callout card-title card-footer"
+      "card-callout card-text card-footer";
+
+    margin: $nx-spacing-md 0;
+    text-align: left;
+    width: 100%;
+  }
+
+  .nx-card-header {
+    -ms-grid-row: 1;
+    -ms-grid-column: 2;
+    grid-area: card-title;
+
+    @include container-vertical;
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl $nx-spacing-md $nx-spacing-xxl;
+  }
+
+  .nx-card__text {
+    -ms-grid-row: 2;
+    -ms-grid-column: 2;
+    grid-area: card-text;
+
+    border-right: 1px solid #d1d6f0;
+    border-left: 1px solid #d1d6f0;
+    margin: 0;
+    padding: 0 $nx-spacing-xxl;
+  }
+
+  .nx-card__call-out {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: card-callout;
+    -ms-grid-column-align: center;
+    justify-self: center;
+
+    margin: 0;
+    padding: 0 $nx-spacing-xxl 0 0;
+  }
+
+  .nx-card-footer {
+    -ms-grid-row-align: center;
+    align-self: center;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 3;
+    grid-area: card-footer;
+    -ms-grid-column-align: start;
+    justify-self: start;
+
+    border: none;
+    margin: 0;
+    padding: 0 0 0 $nx-spacing-xxl;
+    // necessary to make the link wrap
+    max-width: 150px;
+  }
+}
+
+// Styles for the alternate vertical style
+.nx-card-container--row {
+  .nx-card--vertical-alt {
+    .nx-card-content {
+      align-items: center;
+      display: flex;
+      -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+      column-gap: 24px;
+    }
+
+    .nx-card__call-out {
+      @include container-horizontal;
+      display: inline-block;
```

Comment:
This appears to have no effect.  Even the Computed tab in the dev tools still shows `block`

Reply:
> This appears to have no effect. Even the Computed tab in the dev tools still shows `block`

Yeah the `display` property has no effect on flex objects: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/33c3e47c8f0339b4542500c44449588376de3360

Reply:
> Yeah the display property has no effect on flex objects: 33c3e47

There was another one a few lines down.

Reply:
> > Yeah the display property has no effect on flex objects: [33c3e47](https://github.com/sonatype/sonatype-react-shared-components/commit/33c3e47c8f0339b4542500c44449588376de3360)
> 
> There was another one a few lines down.

https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/0fc435f252062d5f621c320a909a855baa3c26ff